client/layout/guided-tours/tours/checklist-about-page-tour.js

```diff
@@ -26,31 +25,51 @@ export const ChecklistAboutPageTour = makeTour(
 	<Tour name="checklistAboutPage" version="20171205" path="/non-existent-route" when={ noop }>
 		<Step
 			name="init"
-			placement="right"
+			placement="beside"
+			arrow="left-top"
+			target="side-menu-page"
 			style={ {
 				animationDelay: '0.7s',
 			} }
 		>
 			<p>
 				{ translate(
-					'The About Page is often the most visited page on a site. ' +
+					'Your site comes with an About Page to help get you started. Click on ' +
+						'{{b}}Site Pages{{/b}} to see all the pages on our site.',
+					{
+						components: { b: <strong /> },
+					}
+				) }
+			</p>
+			<Continue target="side-menu-page" step="choose-page" click hidden />
+		</Step>
+
+		<Step name="choose-page" target="page-about" arrow="top-left" placement="below">
+			<p>
+				{ translate( 'Now click {{b}}About{{/b}} so we can start editing this page.', {
+					components: { b: <strong /> },
+				} ) }
+			</p>
+			<Continue target="page-about" step="about-page" click hidden />
+		</Step>
+
+		<Step name="about-page" placement="right">
+			<p>
+				{ translate(
+					'Welcome to your About Page — this is often the most visited page on a site. ' +
 						'You might find that it never feels quite done - that’s OK. ' +
 						'This is the internet and we can update it as many times as we want. ' +
 						'The key is to just get it started.'
 				) }
 			</p>
 			<p>
 				{ translate(
-					'Let’s start by changing the default text with an introduction. ' +
-						'Here are some questions to help you out: Who are you and where are you based? ' +
-						'Why did you start this site? ' +
+					'Let’s change the default text with a new introduction. Here are some questions ' +
```

Comment:
Hi! I've found a possible matching string that has already been translated **1** time:
`translate( 'Let’s start by changing the default text with an introduction. Here are some questions to help you out: Who are you and where are you based? Why did you start this site? What can visitors expect to get out of it?' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).